Source/ACE/Network/GameAction/Actions/GameActionUseItem.cs

```diff
@@ -0,0 +1,66 @@
+﻿using ACE.Common.Extensions;
+using ACE.Database;
+using ACE.Entity;
+using ACE.Entity.Enum;
+using ACE.Managers;
+using ACE.Network.GameEvent;
+using ACE.Network.GameEvent.Events;
+using ACE.Network.GameMessages.Messages;
+using System;
+using System.Collections.Generic;
+
+namespace ACE.Network.GameAction.Actions
+{
+    public static class GameActionUseItem
+    {
+        [GameAction(GameActionType.EvtInventoryUseEvent)]
+        public static void Handle(ClientMessage message, Session session)
+        {
+            // this is specifically for lifestones:
+            // get the object id
+            uint fullId = message.Payload.ReadUInt32();
+            var objectId = new ObjectGuid(fullId);
+
+            // get the worldobject from the id, if it exists
+            AceObject landBlockObject = DatabaseManager.World.GetObjectsByLandblock(session.Player.Location.LandblockId.Landblock).Find(o => o.AceObjectId == objectId.Full);
```

Comment:
I applaud the effort here.  It's the best you could do with what you have.  We really just need that action queue - stuff like this isn't going to fly in the long run.